src/reconstruction/CoreModel.jl

```diff
@@ -305,7 +305,6 @@ end
 @_change_bounds_fn CoreModel Int inplace begin
     isnothing(lower) || (model.xl[rxn_idx] = lower)
     isnothing(upper) || (model.xu[rxn_idx] = upper)
-    nothing
```

Comment:
why remove this? we actually added it quite lately

Reply:
No idea, added it back